action.yml

```diff
@@ -234,6 +234,15 @@ runs:
           fi
         fi
 
+    - name: Install MacOS clang dependencies
+      if: runner.os == 'macOS'
+      shell: bash
+      continue-on-error: true
+      run: |
+        brew install llvm@${{ inputs.version }}
```

Comment:
Should we consider adding any applicable [`brew install` CLI options](https://docs.brew.sh/Manpage#install-options-formulacask-) here? I don't see anything of importance, but I just wanted to mention that we have options.

I think the default behavior is suitable for our usage, but maybe we can better integrate caching in the future. 🤷🏼‍♂️